base.php

```diff
@@ -1714,6 +1714,11 @@ function run() {
 			$preflight=
 				isset($this->hive['HEADERS']['Access-Control-Request-Method']);
 		}
+
+		// Check if the PATH contains only alphanumeric characters and slashes; if not, trigger a 404 error.
+		if (preg_replace('/[^a-zA-Z0-9\/-]/', '', $this->hive['PATH']) !== $this->hive['PATH'])
```

Comment:
Je pense qu'il faut rajouter le charactere `_` à la liste autorisée

Reply:
@wishgroupe C'est fait, sachant aussi que par défaut, il n'y en as pas dans le routes.ini. 
Les underscords peuvent être quand même présent, dans les variables dynamiques.

Question: on ferais pas plus un str_contains($this->hive['path'], '%0A') a la place de la regex ? J'ai juste peur qu'on passe à coté d'un caractère.